### ISSUE TITLE
`cohere.rerank`: Documents as list of strings are unsupported

### ISSUE THREAD

**Issue by d-e-h-i-o** [cohere-ai/cohere-typescript#106]
The docs [specify that documents can also be passed as list of strings](https://docs.cohere.com/reference/rerank-1), and the default example also uses that format. However, the library doesn't support that format.

Version: `7.4.1`

Reproducible sample:
```javascript
import "https://deno.land/x/xhr@0.1.0/mod.ts"
import { CohereClient } from "https://esm.sh/cohere-ai@7.4.1";

export const cohere = new CohereClient({
  token: <token>,
});

const res = await cohere.rerank({
  query: "What is the capital of the United States?",
  model: "rerank-multilingual-v2.0",
  documents: [
    'The Commonwealth of the Northern Mariana Islands is a group of islands in the Pacific Ocean. Its capital is Saipan.',
    'fewa',
    'Capital punishment (the death penalty) has existed in the United States since beforethe United States was a country. As of 2017, capital punishment is legal in 30 of the 50 states.'
  ],
});
```

Throws the following error:
```
error: Uncaught Error: documents -> [0]: Expected object. Received "The Commonwealth of the Northern Mariana Islands is a group of islands in the Pacific Ocean. Its capital is Saipan.".; documents -> [1]: Expected object. Received "fewa".
    at https://esm.sh/v135/cohere-ai@7.4.1/denonext/cohere-ai.mjs:12:1604
    at Generator.next (<anonymous>)
    at f (https://esm.sh/v135/cohere-ai@7.4.1/denonext/cohere-ai.mjs:12:1013)
```

**Comment by billytrend-cohere** [comment]
Thanks for reporting this! I'm sure you know that for now, you can use `{ "text": "" }` instead but I'll fix this today.

**Comment by billytrend-cohere** [comment]
Should be fixed in 7.4.3 Thanks again @d-e-h-i-o
